### ISSUE TITLE
Can't install on OpenSuse Tumbleweed (qt3 support library missing)

### ISSUE THREAD

**Issue by ilovefreesw** [mviereck/kaptain#4]
![image](https://user-images.githubusercontent.com/77317006/136644650-b0b689db-660d-4d8d-bb21-8d3a7ff99492.png)

**Comment by mviereck** [comment]
Yes, the libraries for qt3 support are disappearing. I have the same issue on recently published debian bullseye.

If you want to use it for `x11docker-gui`, I suggest to pull image `x11docker/kaptain` instead.

I could try to statically link the qt3 support libraries. But in fact I consider to give up kaptain.
It would make most sense to port kaptain to qt5. But I am not familiar with qt or c++.

**Comment by image28** [comment]
Responding to an old post. I'll have a go porting it to qt4 with the qt3to4 app tonight. Used to use this software decades ago and would like to improve it and create a derivative works for my own purposes, will let you know how I go tomorrow morning.

**Comment by image28** [comment]
Ran into issues compiling qt4 with current compilers last night. Got most of the library's (shared object files) compiled. Possibly even enough to make a binary of Kaptain. Will test that now. But also setting up multiple virtual machine from old linux cdroms to run the qt3to4 source code porting utility, then will look into moving the code up to version 5 and finally 6.

**Comment by mviereck** [comment]
> will look into moving the code up to version 5 and finally 6.

That sounds really great! Did you make further progress? Do you succeed?
